src/styles/product_card.scss

```
@@ -0,0 +1,115 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  background-color: #fff;
+  min-width: 1024px;
+  max-width: 1200px;
+  margin: auto;
+}
+
+.card {
+  font-family: "Roboto", sans-serif;
+  max-width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border-radius: 5px ;
+  margin-right: 24px;
+  margin-left: 24px;
+  margin-top: 48px;
+  padding-left: 16px;
+  padding-right: 16px;
+  text-align: center;
+  border: 1px solid #f3f3f3;
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+  &__text {
+    text-align: left;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    max-width: 166px;
+    padding-top: 36px;
+    padding-bottom: 4px;
+  }
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: #616070;
+    padding-bottom: 16px;
+  }
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 24px;
+  }
+  &__reviews-stars {
+    display: flex;
+  }
+  .stars {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+  &__reviews-text {
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+    display: flex;
+    align-items: flex-end;
+    text-align: right;
+  }
+  &__reviews-stars :nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+  }
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 16px;
+  }
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+  &__price-number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
+  &__button {
```

```suggestion
  &__price-text {
    font-size: 12px;
    line-height: 18px;
    color: #616070;
  }
  
  &__price-number {
    font-weight: 700;
    font-size: 16px;
    line-height: 18px;
    color: #060b35;
  }
  
  &__button {
```
put some blank lines to separate scss blocks